### PR TITLE
OpenMC: new version 0.13.0

### PR DESCRIPTION
**Pull request by paulromano**
This PR adds version 0.13.0 for packages openmc and py-openmc.

### CODE REVIEW

File: var/spack/repos/builtin/packages/openmc/package.py [hunk 1]
```diff
@@ -8,18 +8,22 @@
...
+       programming model."""
 
     homepage = "https://docs.openmc.org/"
-    url = "https://github.com/openmc-dev/openmc/tarball/v0.12.2"
+    url = "https://github.com/openmc-dev/openmc/tarball/v0.13.0"
     git = "https://github.com/openmc-dev/openmc.git"
 
     version('develop', branch='develop', submodules=True)
     version('master', branch='master', submodules=True)
+    version('0.13.0', tag='v0.13.0', submodules=True)
```
Comment (sethrj) [inline var/spack/repos/builtin/packages/openmc/package.py]: Can you add the git `commit='...'` argument to ensure reproducibility and improve security? (Tags can be deleted, overridden, etc.) If you're bored you can also add them to the other versions :)

File: var/spack/repos/builtin/packages/py-openmc/package.py [hunk 2]
```diff
@@ -7,18 +7,22 @@
...
+       programming model."""
 
     homepage = "https://docs.openmc.org/"
-    url = "https://github.com/openmc-dev/openmc/tarball/v0.12.2"
+    url = "https://github.com/openmc-dev/openmc/tarball/v0.13.0"
     git = "https://github.com/openmc-dev/openmc.git"
 
     version('develop', branch='develop')
     version('master', branch='master')
+    version('0.13.0', tag='v0.13.0', submodules=True)
```
Comment (sethrj) [inline var/spack/repos/builtin/packages/py-openmc/package.py]: Same here

File: var/spack/repos/builtin/packages/openmc/package.py [hunk 3]
```diff
@@ -8,18 +8,22 @@
...
+       subcritical multiplication calculations on models built using either a
+       constructive solid geometry or CAD representation. OpenMC supports both
+       continuous-energy and multigroup transport. The continuous-energy particle
+       interaction data is based on a native HDF5 format that can be generated from ACE
+       files produced by NJOY. Parallelism is enabled via a hybrid MPI and OpenMP
+       programming model."""
 
     homepage = "https://docs.openmc.org/"
-    url = "https://github.com/openmc-dev/openmc/tarball/v0.12.2"
+    url = "https://github.com/openmc-dev/openmc/tarball/v0.13.0"
```
Comment (sethrj) [inline var/spack/repos/builtin/packages/openmc/package.py:21]: The `url` field is used as a template for downloads and only needs updating if the format changed. It's fine to leave in, but you can skip it on future PRs.

File: var/spack/repos/builtin/packages/py-openmc/package.py [hunk 4]
```diff
@@ -38,7 +42,7 @@ class PyOpenmc(PythonPackage):
         )
 
     depends_on('git', type='build')
-    depends_on('python@3.5:', type=('build', 'run'))
+    depends_on('python@3.6:', type=('build', 'run'))
```
Comment (sethrj) [inline var/spack/repos/builtin/packages/py-openmc/package.py]: The dependency here should be conditional on the version:
```suggestion
    depends_on('python@3.6:', type=('build', 'run'), when='@0.13:')
    depends_on('python@3.5:', type=('build', 'run'), when='@:0.12')
```

**Review by sethrj — CHANGES_REQUESTED** [review]

**Comment by paulromano** [comment]
Thanks for reviewing @sethrj! Just updated the `version(...)` to use git commit so hopefully good to go now

**Review by sethrj — APPROVED** [review]
Excellent!

**Pull request merged**